### PR TITLE
Added copying of map to the log file recorded

### PR DESCRIPTION
**Pull request by skwirskj**
- Saves a copy of the map file in the data logging directory under the same name of the directory or walk file
- Adds the path of the map directory/file to the metadata

```json
{
  "bosdyn.api version": "3.3.2",
  "date": "2024-02-09T14:10:55.766229",
  "rgb_sources": [
    "hand_color_image",
    "back_fisheye_image",
    "frontleft_fisheye_image",
    "frontright_fisheye_image",
    "left_fisheye_image",
    "right_fisheye_image"
  ],
  "depth_sources": [
    "hand_depth",
    "frontleft_depth_in_visual_frame",
    "frontright_depth_in_visual_frame"
  ],
  "period": null,
  "map_path": "data/click_map_data_1707505855/downloaded_graph"
  ```

### CODE REVIEW

File: src/conq/data_recorder.py
Comment (dcolli23) [inline src/conq/data_recorder.py:174]: Since you're taking in a `Path` object and using Python >= 3.8, consider doing the following:

```python
		if map_path.exists():
            print(f"MAP PATH: {map_path}")
            dst = self.root / map_path.name
            if map_path.is_dir():
                shutil.copytree(map_path, dst)
            else:
                shutil.copyfile(map_path, dst)
			self.metadata['map_path'] = dst.as_posix()
        else:
            print(f"WARNING from data_recorder.py: map path {map_path} does not exist! Not adding path to metadata.")
```

**Review by dcolli23 — COMMENTED** [review]
Let me know if you don't think the change I mentioned is suitable. Other than that, looks good

File: src/conq/data_recorder.py
Comment (skwirskj) [reply src/conq/data_recorder.py:172]: Good suggestion! Done in https://github.com/UM-ARM-Lab/conq_python/pull/35/commits/6b4d3c476b68ed75402a03d105fce243aa1a49d1

**Review by dcolli23 — APPROVED** [review]
Whoops! Sorry, didn't realize you were waiting on approval for this. Feel free to squash and merge

**Pull request merged**
